scanners/sslyze.py

```diff
@@ -589,7 +686,7 @@ def queue(command):
             return None, None, None, None, None, None, None
 
     # Initialize commands and result containers
-    sslv2, sslv3, tlsv1, tlsv1_1, tlsv1_2, tlsv1_3, certs = None, None, None, None, None, None
+    sslv2, sslv3, tlsv1, tlsv1_1, tlsv1_2, tlsv1_3, certs, reneg = None, None, None, None, None, None, None, None
```

Comment:
This can be simplified to:

`sslv2 = sslv3 = tlsv1 = tlsv1_1 = tlsv1_2 = tlsv1_3 = certs = reneg = None`

Reply:
:+1: